src/style.css

```diff
@@ -1,3 +1,91 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14.06px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.header {
+  padding: 0 50px;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.img {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  margin-left: 50px;
+  align-items: center;
+}
+
+.container {
+  width: 1200px;
+  height: 60px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav ul {
+  display: flex;
+  text-decoration: none;
+  margin: 0 auto;
+  padding: 0;
+  align-items: center;
+  list-style: none;
+}
+
+.laptops {
+  color: #00ACDC;
+}
+
+nav a {
```

Comment:
<img width="1121" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/861208db-df09-4649-a8c5-0fafb093b9ef">
follow the checklist